docs/src/design/decoder/constraints.md

```diff
@@ -168,8 +168,8 @@ $$
 Using the above definitions, we can describe the constraint for computing block hashes as follows:
 
 > $$
-p_0' \cdot (u_{join} + u_{split} + u_{loop} + u_{span} + u_{respan} + u_{end} + \\
-1 - (f_{join} + f_{split} + f_{loop} + f_{span} + f_{respan} + f_{end})) = p_0
+p_0' \cdot (u_{ctrli} + u_{span} + u_{respan} + u_{end} + \\
+1 - (f_{ctrli} f_{span} + f_{respan} + f_{end})) = p_0
```

Comment:
I think there is a "+" missing after "f_{ctrli}".

Reply:
updated